am/templates/api/api-configmap.yaml

```diff
@@ -12,9 +12,33 @@ metadata:
     helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
 data:
   gravitee.yml: |
-    {{- if .Values.api.ssl.enabled }}
+    {{- if .Values.api.http }}
     jetty:
```

Comment:
`jetty` section should always be available, because if a customer uses the `Values.api.ssl.enabled` before, it will not work anymore